dipy/viz/ui.py

```diff
@@ -2161,3 +2163,185 @@ def handle_move_callback(self, i_ren, obj, slider):
         self.move_handle(click_position=click_position)
         i_ren.force_render()
         i_ren.event.abort()  # Stop propagating the event.
+
+
+class ImageHolder(UI):
```

Comment:
`ImageHolder2D` probably be a better name.

Reply:
I don't love "holder." Maybe `ImageContainer2D`?